app/views/teams/show.html.erb

```diff
@@ -18,6 +18,33 @@
   <dd><%= @team.description %></dd>
 </dl>
 
+<table class="table table-striped">
+  <thead>
+    <tr>
+    </tr>
+  </thead>
+  <tbody>
+    <% @team.members.each do |member| %>
+      <tr>
+        <td>
+            <%= member.id%> <%= member.first_name%> <%= member.last_name%>
+            
+            <% if @team.owners.include? member %>
+            <%= link_to t('.delete_ownership', :default => t("helpers.links.delete_ownership")),
+            delete_ownership_team_path(@team, :team_member => member.id), :method => 'post', :class => 'btn btn-default btn-xs' %>
```

Comment:
Consider using a different naming like `team_member_id` or `member_id`

Reply:
Did you define the required parameters for the `delete_ownership` route somewhere, e.g. to throw an exception when a invalid parameter type is given in a specific view?

Reply:
wrong params throw an exception #dontscrewmyroutes

Reply:
should be stylize #hashtags in comments like _#so_ or *#so*? opinions please!